commands/download.bee.inc

```diff
@@ -298,9 +294,20 @@ function download_bee_git_info($project) {
     )));
 
     if ($answer) {
+      $repo_info = FALSE;
+      if ($curl_handle = curl_init("$api_url/$organisation/$project")) {
+        curl_setopt($curl_handle, CURLOPT_USERAGENT, 'Bee');// github nag @todo constant
```

Comment:
how about 'Bee - the command line tool for Backdrop CMS' ? It works (I've tested that variant) and is clear what is and what it relates to if they ever try to block dodgy user agents, then this should be descriptive enough not to get blocked because they don't know what it is.

Were you thinking about a global constant or a constant at the command level. I'm easy either way.

Reply:
> Were you thinking about a global constant or a constant at the command level

Interesting point... I was thinking about a command level constant, AFAIK the download command is the only one, that needs cURL. The rest seems to use core functions.